src/api/uploads/__tests__/MultiputPart-test.js

```diff
@@ -184,46 +183,24 @@ describe('api/uploads/MultiputPart', () => {
             });
         });
 
-        withData(
-            [
-                [
-                    {
-                        offset: 1,
```

Comment:
This one seems to have 2 objects

Reply:
It was somewhat weird. 

When I logged the contents of the previous test, it came out as one object, and when I tried including the data as two separate objects the test failed.

Reply:
Lemme re-try that.